specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2500,16 +2500,27 @@
                 },
                 "allowedSourceAddressPrefix": {
                     "type": "string",
-                    "description": "Source of the allowed traffic. If omitted, the request will be for the source IP address of the initiate request."
+                    "description": "Mutually exclusive with the \"allowedSourceAddressPrefixes\" parameter. Should be an IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\". If both are omitted, the request will be for the source IP address of the initiate request."
+                },
+                "allowedSourceAddressPrefixes": {
+                    "type": "array",
+                    "description": "Mutually exclusive with the \"allowedSourceAddressPrefix\" parameter. If both are omitted, the request will be for the source IP address of the initiate request.",
+                    "items": {
+                        "type": "string",
+                        "description": "IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"."
+                    }
+                },
+                "duration": {
+                    "type": "string",

```

Comment:
```suggestion
                    "type": "string",
                    "format": "iso8601",
```

Reply:
Seems like this format is not support by some of the tools and caused the validation to fail.